### PR TITLE
Vendor Device Storage and Operational Disclosures

### PR DESCRIPTION
**Pull request by anderagakura**
Vendor Device Storage and Operational Disclosures that supersedes the current Device Storage Access and Disclosure specification

### CODE REVIEW

File: TCFv2/Vendor Device Storage & Operational Disclosures.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+The TCF registration process requires Vendors to provide a secure URL to a JSON resource that conforms to the content and structure specified below. The Managing Organisation publishes the URL on the GVL along with other Vendor registration information.
+
+The JSON contains two types of information, disclosures related to device storage access and duration (the _Disclosures_ object and attributes) and the web domains the Vendor uses (the _Domains_ object and attributes). Both are required, though not all information within each object is required. See the tables below. 
+
+### Disclosures Object
+
+Vendors MUST publish granular disclosures for web-based storage - ‘cookie’ and ‘web’ mechanism types. Vendors MAY provide ’app’ storage type disclosures, but are not required to do so. 
+
+<table>
```
Comment (tla-sirdata) [inline TCFv2/Vendor Device Storage & Operational Disclosures.md:47]: Some fields are specified as optional/required in both "scope" and "description" columns, which can be confusing. It should only be described in the "scope" column (like `cookieRefresh`)

Field names should be highlighted.

File: TCFv2/Vendor Device Storage & Operational Disclosures.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+}
+
+````
+AdTech123 publishes this information at https://www.adtech123.com/path/to/deviceStorage.json, and provides this URL to the TCF during the registration process. 
+
+### Domains Object
+
+Vendors MUST publish the domains they use for collecting and processing personal data in the context of their TCF registration. Vendors MUST NOT include Publishers’ delegated domains or subdomains they may use.
+
+<table>
```
Comment (tla-sirdata) [inline TCFv2/Vendor Device Storage & Operational Disclosures.md:106]: Field names should be highlighted.

File: TCFv2/Vendor Device Storage & Operational Disclosures.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
+# Vendor Device Storage & Operational Disclosures
+
+ **IAB Europe Transparency & Consent Framework**
+
+**Final v.2.0 | August 2019, Updated September 2021**
```
Comment (tla-sirdata) [inline TCFv2/Vendor Device Storage & Operational Disclosures.md:5]: Date must be updated.

**Review by tla-sirdata — CHANGES_REQUESTED** [review]

File: TCFv2/Vendor Device Storage & Operational Disclosures.md (hunk 1 above)
Comment (anderagakura) [reply TCFv2/Vendor Device Storage & Operational Disclosures.md:47]: Quick question : Why should we highlight the field names while in the other pages (example [here](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20Consent%20string%20and%20vendor%20list%20formats%20v2.md)), it's not the case?

Comment (tla-sirdata) [reply TCFv2/Vendor Device Storage & Operational Disclosures.md:47]: It's just to make it easier to read and help understand that they are "values" and not descriptive text. Thanks

Comment (anderagakura) [reply TCFv2/Vendor Device Storage & Operational Disclosures.md:47]: Done [here](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/pull/313/commits/206bfa01eee11f008882676c37d757bc98a83f98)

File: TCFv2/Vendor Device Storage & Operational Disclosures.md (hunk 2 above)
Comment (anderagakura) [reply TCFv2/Vendor Device Storage & Operational Disclosures.md:106]: Done [here](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/pull/313/commits/206bfa01eee11f008882676c37d757bc98a83f98)

File: TCFv2/Vendor Device Storage & Operational Disclosures.md (hunk 3 above)
Comment (anderagakura) [reply TCFv2/Vendor Device Storage & Operational Disclosures.md]: Done [here](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/pull/313/commits/206bfa01eee11f008882676c37d757bc98a83f98)

**Review by HeinzBaumann — APPROVED** [review]

**Pull request merged**